app-article/src/main/java/top/ruilink/realworld/article/domain/Test.java

```diff
@@ -0,0 +1,48 @@
+package top.ruilink.realworld.article.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Entity
```

Comment:
The class name 'Test' is too generic and does not convey the purpose of the entity. Consider renaming it to something more descriptive that reflects its role in the application.